can-define-class.js

```diff
@@ -74,8 +87,15 @@ function define(Base = Object) {
 
 		constructor(props) {
 			super();
+			Object.defineProperty(this, inSetupSymbol, {
```

Comment:
This doesn't appear to be used for anything at the moment, for something you'll use later?

Reply:
This is used by `can-event-queue`. Setting it here prevents events from being fired during the setup.

Reply:
Ok cool, I would say merge then.